Documentation/HelpInside/KeyboardCommands/Index.rst

```diff
@@ -23,6 +23,20 @@ the guidelines set out in `WAI-ARIA Authoring Practices 1.1. <https://www.w3.org
 *  :kbd:`Right` arrow key will expand the focus if possible.
 *  :kbd:`Left` arrow key will close the focus if possible.
 
+Actions in backend editor
```

Comment:
How about "in backend forms"? Maybe "editor" might make people think it's related to the RTE?